packages/autocomplete-js/src/elements/Input.ts

```diff
@@ -46,6 +48,11 @@ export const Input: AutocompleteElement<InputProps, HTMLInputElement> = ({
         return;
       }
 
+      if (useTrapFocus && event.key === 'Tab') {
+        event.preventDefault();
+        return;
+      }
```

Comment:
This is not the expected behavior, we don't want to prevent <kbd>Tab</kbd> events, but rather to _trap_ the focus within the modal, [as we're doing in DocSearch](https://github.com/algolia/docsearch/blob/bf18c7374e66581bd0b08b19f7a61b4a30bdcef3/packages/docsearch-react/src/useTrapFocus.ts).